pkg/systemd/services/timer-dialog.jsx

```diff
@@ -233,27 +248,58 @@ const CreateTimerDialogBody = ({ owner }) => {
                                         }}
                                         aria-label={_("Repeat")}>
                                 <FormSelectOption value="no" label={_("Don't repeat")} />
+                                <FormSelectOption value="minutely" label={_("Minutely")} />
                                 <FormSelectOption value="hourly" label={_("Hourly")} />
                                 <FormSelectOption value="daily" label={_("Daily")} />
                                 <FormSelectOption value="weekly" label={_("Weekly")} />
                                 <FormSelectOption value="monthly" label={_("Monthly")} />
                                 <FormSelectOption value="yearly" label={_("Yearly")} />
+                                <FormSelectOption value="custom" label={_("Custom")} />
                             </FormSelect>
                         </FormGroup>
                         {repeat == "no" &&
                         <FormGroup label={_("Run at")}>
                             <TimePicker className="create-timer-time-picker specific-no-repeat"
                                         menuAppendTo={() => document.body} time={specificTime} is24Hour onChange={setSpecificTime} />
                         </FormGroup>}
+                        {repeat == "custom" &&
+                        <FormGroup
```

Comment:
You can use `labelIcon` property here, see for example https://www.patternfly.org/v4/components/form

Right now it somehow breaks:
![Screenshot from 2022-07-27 10-19-03](https://user-images.githubusercontent.com/12330670/181198364-a8ba1828-f365-4229-84ab-46e840b1e010.png)

Reply:
Remove the icon as we are switching to helper text